Doc/library/os.rst

```diff
@@ -5555,7 +5555,7 @@ Miscellaneous System Information
 
    .. versionchanged:: 3.13
       If :option:`-X cpu_count <-X>` is given or :envvar:`PYTHON_CPU_COUNT` is set,
-      :func:`cpu_count` returns the overridden value *n*.
+      :func:`cpu_count` returns the override value *n*.
```

Comment:
The language is weird either way, may be 'overriding value'

Reply:
Hi there. Shouldn't it be like "return value" used in these docs?